client/src/Components/BoxBooks/BoxBooksStyles.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components';
+
+export const BoxBooksComponent = styled.div`
+  position:relative;
+  font-family:${(props) => props.theme.fonts.default};
+  color: ${(props) => props.theme.colors.secondary};
+  background: ${(props) => props.theme.backgrounds.darkTransparent};
+  box-shadow: ${(props) => props.theme.shadows.inset};
+  width:40vw;
+  height:30vw;
```

Comment:
Może bardziej responsywne jednostki np. % ?

Reply:
to są jakieś bardziej responsywne niż te zależące wprost od wielkości viewportu? :)  a na poważnie to wydaje mi się, że jak zrobię tutaj height w % to będzie problem, bo obrazek mamy w position absolute i będzie wyjeżdżał albo po prostu nie wiem jak to zrobić to mnie proszę naprowadź...

Reply:
Sorry, jest dobrze, źle przeczytałem